REVEL.pm

```diff
@@ -0,0 +1,143 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2017] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ REVEL
+
+=head1 SYNOPSIS
+
+ mv REVEL.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin REVEL,/path/to/revel/data.tsv.gz
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ adds the REVEL score for missense variants to VEP output.
+
+ Please cite the CADD publication alongside the VEP if you use this resource:
+ https://www.ncbi.nlm.nih.gov/pubmed/27666373
```

Comment:
Still references CADD here; REVEL is https://www.ncbi.nlm.nih.gov/pubmed/27666373

You'll also need to provide instructions on converting and indexing the REVEL data file as it is CSV formatted IIRC?